third_party/blink/renderer/core/input/mouse_event_manager.cc

```diff
@@ -277,7 +277,7 @@ WebInputEventResult MouseEventManager::DispatchMouseEvent(
     auto pos = mouse_event.PositionInRootFrame();
     auto x = (size_t)pos.x();
     auto y = (size_t)pos.y();
-    recordreplay::OnMouseEvent(mouse_event_type.Ascii().c_str(), x, y);
+    recordreplay::OnMouseEvent(mouse_event_type.Utf8().c_str(), x, y);
```

Comment:
q: are we expecting this to ever actually be utf8?

Reply:
I guess not. I thought there would be some odd localization going on, but it turns out that the reason why the strings are generated dynamically is because of platform differences.

They come from input files like [this](https://github.com/replayio/chromium/blob/56587dc14d9787442b3ec53dda4e7f364a62aaeb/third_party/blink/renderer/core/events/event_type_names.json5) and [this](https://github.com/replayio/chromium/blob/1eca5e5f77669751efaccd263849c7a95e0ccd17/third_party/blink/renderer/extensions/chromeos/event_type_chromeos_names.json5).